### PR TITLE
Added CLI and usage instructions

### PR DESCRIPTION
**Pull request by paulkorir**
* CLI command `roman`
* full test coverage
* documentation in README.rst
* updated CHANGES.txt for v3.3
* version bump: v3.3.dev1

### CODE REVIEW

**Comment by paulkorir** [comment]
The drop in coverage is due to the ``if __name__ == "__main__": sys.exit(main())``. ;-)

**Comment by dataflake** [comment]
Hi Paul, please see https://www.zope.org/developer/becoming-a-committer.html and submit a signed contributor agreement before this can be merged.

**Comment by jugmac00** [comment]
Hi @paulkorir 

thanks a lot for your contribution!

> The drop in coverage is due to the if __name__ == "__main__": sys.exit(main()). ;-)

That is perfectly fine!

Just for your info:
For my command line projects, I usually do this `os.system` trick to get 100% :-)

https://github.com/jugmac00/hibpcli/blob/54c961d32386f8ced1f05ad8bc3bd73fd728569a/tests/test_main_program.py#L4-L10

**Comment by paulkorir** [comment]
> Hi Paul, please see https://www.zope.org/developer/becoming-a-committer.html and submit a signed contributor agreement before this can be merged.

I've sent the signed agreement.

**Comment by dataflake** [comment]
Thanks, the Plone Foundation folks will review and once it's been accepted your status will change to `Member` here. I'll review/merge after that.

File: setup.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 setup(
     name='roman',
-    version='3.3.dev0',
+    version='3.3.dev1',
```
Comment (dataflake) [inline setup.py:8]: Please remove the version change. The person who does a release makes all necessary changes.

File: src/roman.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```
Comment (dataflake) [inline src/roman.py:1]: Please remove this import and all references to `print_function`, just use the `print` function directly, which is fully supported in the lowest Python version this package supports (2.7)

Comment (jamadden) [reply src/roman.py:1]: `print` is only a function in Python 2.7 with that import. Otherwise it's still a statement.

Comment (dataflake) [reply src/roman.py:1]:
```python
$ bin/python2.7
Python 2.7.17 (default, Jun 18 2020, 16:46:00)
[GCC 4.2.1 Compatible Apple LLVM 11.0.3 (clang-1103.0.32.62)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> print('foo')
foo
>>>
```

Comment (jamadden) [reply src/roman.py:1]: That's a common misunderstanding (and a trick commonly used in doctests!). A one-element tuple *looks* like a function call and appears to behave like one, but as soon as you add other elements, you see the difference. (The lack of a return value is also a give-away.)

```pycon
Python 2.7.14 (default, Mar 14 2018, 13:36:31)
[GCC 7.3.1 20180303 (Red Hat 7.3.1-5)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> print('foo')
foo
>>> print('foo', 'bar')
('foo', 'bar')
>>> print

>>>
```

compared to
```pycon
Python 3.8.3 (default, May 16 2020, 08:39:55)
[Clang 11.0.3 (clang-1103.0.32.29)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> print('foo')
foo
None
>>> print('foo', 'bar')
foo bar
None
>>> print
<built-in function print>
>>>
```

Comment (dataflake) [reply src/roman.py:1]: You're right. Learn something new every day 😄

**Comment by dataflake** [comment]
I'll just do the last cleanup on `master` myself

**Comment by dataflake** [comment]
I just released roman version 3.3 with your changes. Thanks!

**Pull request merged**